test/source/tests/browser-unit-tests/unit-ContactStore.js

```diff
@@ -371,57 +299,20 @@ BROWSER_UNIT_TEST_NAME(`ContactStore stores postfixed fingerprint internally for
   // extract the entity directly from the database
   const entityFp = '16BB407403A3ADC55E1E0E4AF93EEC8FB187C923-X509';
   const fingerprint = '16BB407403A3ADC55E1E0E4AF93EEC8FB187C923';
-  const longid = 'X509-MIGiMIGNMQswCQYDVQQGEwJJVDEQMA4GA1UECAwHQmVyZ2FtbzEZMBcGA1UEBwwQUG9udGUgU2Fu' +
-    'IFBpZXRybzEjMCEGA1UECgwaQWN0YWxpcyBTLnAuQS4vMDMzNTg1MjA5NjcxLDAqBgNVBAMMI0FjdGFsaXMgQ2xpZW50IE' +
-    'F1dGhlbnRpY2F0aW9uIENBIEcyAhBj9wJecA85RTAfsvulZ0+E';
   const entity = await new Promise((resolve, reject) => {
     const req = db.transaction(['pubkeys'], 'readonly').objectStore('pubkeys').get(entityFp);
     ContactStore.setReqPipe(req, resolve, reject);
   });
   if (entity.fingerprint !== entityFp) {
     throw Error(`Failed to extract pubkey ${fingerprint}`);
   }
-  const [contactByLongid] = await ContactStore.get(db, [longid]);
-  if (contactByLongid.pubkey.id !== fingerprint) {
-    throw Error(`Failed to extract pubkey ${fingerprint}`);
-  }
   const [contactByEmail] = await ContactStore.get(db, [email]);
   if (contactByEmail.pubkey.id !== fingerprint) {
     throw Error(`Failed to extract pubkey ${fingerprint}`);
   }
   return 'pass';
 })();
 
-BROWSER_UNIT_TEST_NAME(`ContactStore searches S/MIME Certificate by PKCS#7 message recipient`);
```

Comment:
Is this usecase covered elsewhere, or not needed?

Reply:
Search by longid is no longer required anywhere in the system. We're now loading all sender's keys and only use them to verify. This should also be the case for S/MIME messages (though verification isn't yet implemented for them)

Reply:
Got it